checkov/kustomize/runner.py

```diff
@@ -232,23 +261,32 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
                         self.kustomizeProcessedFolderAndMeta[filePath]['overlay_name'] = checkovKustomizeEnvNameByPath
                         logging.warning(f"Could not confirm base dir for Kustomize overlay/env. Using {checkovKustomizeEnvNameByPath} for Checkov Results.")
             
+
+                if self.templateRendererCommand == "kubectl":
+                    templateRenderCommandOptions = "kustomize"
+                if self.templateRendererCommand == "kustomize":
+                    templateRenderCommandOptions = "build"
+                    
                 # Template out the Kustomizations to Kubernetes YAML
                 try:
-                
-                    proc = subprocess.Popen([self.kustomize_command, 'build', filePath], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
+                    proc = subprocess.Popen([self.templateRendererCommand, templateRenderCommandOptions, filePath], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
                     o, e = proc.communicate()
                     logging.info(
-                        f"Ran {self.kustomize_command} command to build Kustomize output. DIR: {filePath}. TYPE: {self.kustomizeProcessedFolderAndMeta[filePath]['type']}.")
+                        f"Ran {self.templateRendererCommand} to build Kustomize output. DIR: {filePath}. TYPE: {self.kustomizeProcessedFolderAndMeta[filePath]['type']}.")
 
                 except Exception as e:
                     logging.warning(
-                        f"Error build Kustomize output at dir: {filePath}. Error details: {str(e, 'utf-8')}")
+                        f"Error building Kustomize output at dir: {filePath}. Error details: {str(e, 'utf-8')}")
                     continue
 
                 if self.kustomizeProcessedFolderAndMeta[filePath]['type'] == "overlay":
-                    basePathParents = pathlib.Path(self.kustomizeProcessedFolderAndMeta[filePath]['calculated_bases']).parents
-                    mostSignificantBasePath = "/" + basePathParents._parts[-3] + "/" + basePathParents._parts[-2] + "/" + basePathParents._parts[-1]
-                    envOrBasePathPrefix = mostSignificantBasePath + "/" + str(self.kustomizeProcessedFolderAndMeta[filePath]['overlay_name'])
+                    if not 'calculated_bases' in self.kustomizeProcessedFolderAndMeta[filePath]:
```

Comment:
```suggestion
                    if 'calculated_bases' not in self.kustomizeProcessedFolderAndMeta[filePath]:
```
both ways are acceptable, but this is the most common and preferred one

Reply:
Corrected! 👍